conformance/conformance_test.cc

```diff
@@ -1225,6 +1241,34 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
   ExpectParseFailureForJson(
       "OneofFieldDuplicate",
       R"({"oneofUint32": 1, "oneofString": "test"})");
+  // Ensure zero values for oneof make it out/backs.
+  {
+    TestAllTypes message;
+    message.set_oneof_uint32(0);
+    RunValidProtobufTest(
+        "Oneof_Zero_uint32", message, "oneof_uint32: 0");
```

Comment:
Sure, what format would you like?  Just drop the underscores?  OneofZeroUint32, etc.?